intellij/src/org/scalafmt/intellij/ScalafmtAction.scala

```diff
@@ -112,13 +112,16 @@ class ScalafmtAction extends AnAction {
       globalConfig = getConfigFileInPath(homeDir)
       configFile <- localConfig.orElse(globalConfig)
       config <- {
-        val x = StyleCache.getStyleForFile(configFile)
-        if (x.isEmpty) {
-          displayMessage(event,
-                         s"Failed to read $configFile",
-                         MessageType.WARNING)
+        val x = StyleCache.getStyleForFileOrError(configFile)
+        x match {
+          case Left(e:Throwable) => {
+            displayMessage(event,
+              s"Failed to read $configFile" + e.getMessage,
```

Comment:
No worries, it's tricky to run the plugin. I hope #399 will simplify the setup.

Just add a dot at the end in `$configFile. "`